docs/WatchPlugins.md

```diff
@@ -221,8 +228,16 @@ Any key not used by built-in functionality can be claimed, as you would expect.
 
 Should your plugin attempt to overwrite a reserved key, Jest will error out with a descriptive message, something like:
 
-> Watch plugin YourFaultyPlugin attempted to register key `q`, that is reserved internally for quitting watch mode. Please change the configuration key for this plugin.
+:::caution
```

Comment:
This is an error message, so code block would make more sense.

```
    ```bash
    [message goes here]
    ```
```